src/software/telemetry/src/structures.rs

```diff
@@ -55,20 +71,7 @@ pub enum TelemetryMessage {
         blower_rpm: u8,
         battery_level: u8,
     },
-    MachineStateSnapshot {
-        version: String,
-        device_id: String,
-        cycle: u32,
-        peak_command: u8,
-        plateau_command: u8,
-        peep_command: u8,
-        cpm_command: u8,
-        previous_peak_pressure: u8,
-        previous_plateau_pressure: u8,
-        previous_peep_pressure: u8,
-        current_alarm_codes: Vec<u8>,
-        previous_alarm_codes: Vec<u8>,
-    },
+    MachineStateSnapshot(MachineStateSnapshot),
```

Comment:
Maybe I should do this change first in #193 so it is consistent and then you rebase on it when it is merged?

Reply:
Yep, we could do that

Reply:
Cool! I added this change in #193

Reply:
alright, waiting for it to be merged so I can rebase

Reply:
It's merged